src/main/java/uk/gov/hmcts/ccd/auditlog/aop/AuditAspect.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.ccd.auditlog.aop;
+
+import org.apache.commons.lang.StringUtils;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.reflect.MethodSignature;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.expression.AnnotatedElementKey;
+import org.springframework.expression.EvaluationContext;
+import org.springframework.expression.spel.SpelEvaluationException;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.auditlog.LogAudit;
+
+import java.lang.reflect.Method;
+import java.util.List;
+
+@Aspect
+@Component
+public class AuditAspect {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AuditAspect.class);
+
+    private static final String RESULT_VARIABLE = "result";
+
+    private ExpressionEvaluator evaluator = new ExpressionEvaluator();
+
+    @Around("@annotation(logAudit)")
+    public Object audit(ProceedingJoinPoint joinPoint, LogAudit logAudit) throws Throwable {
+        Object result = null;
+        try {
+            result = joinPoint.proceed();
+            return result;
+        } catch (Exception ex) {
+            throw ex;
```

Comment:
there is a logger "LOG" it would be good if we log the trace and a message.  We know where it is coming from

Reply:
Controller advice logs the same. Actually no need to catch and throw here.